plugins/examples/tables/src/index.tsx

```diff
@@ -0,0 +1,62 @@
+import { registerResourceTableColumnsProcessor } from '@kinvolk/headlamp-plugin/lib';
+import { ActionButton } from '@kinvolk/headlamp-plugin/lib/CommonComponents';
+import Pod from '@kinvolk/headlamp-plugin/lib/K8s/pod';
+import { Menu, MenuItem, Typography } from '@material-ui/core';
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+
+function ContextMenu(props: { pod: Pod }) {
+  const { pod } = props;
+  const [anchorEl, setAnchorEl] = React.useState(null);
+  const history = useHistory();
+
+  function handleClick(event: any) {
+    setAnchorEl(event.currentTarget);
+  }
+
+  function handleClose() {
+    setAnchorEl(null);
+  }
+
+  return (
+    <>
+      <ActionButton
+        description="Our action button"
+        icon="mdi:dots-vertical"
+        onClick={handleClick}
+      />
+      <Menu anchorEl={anchorEl} open={Boolean(anchorEl)} onClose={handleClose}>
+        <MenuItem
+          onClick={() => {
+            history.push(pod.getDetailsLink());
+          }}
+        >
+          <Typography>Details</Typography>
+        </MenuItem>
+        <MenuItem
+          onClick={() => {
+            alert('Boom! Pod deleted! (just kidding)');
+            handleClose();
+          }}
+        >
+          <Typography>Delete</Typography>
+        </MenuItem>
+      </Menu>
+    </>
+  );
+}
+
+registerResourceTableColumnsProcessor(function ageRemover({ id, columns }) {
+  console.debug('Resource table to process:', id);
+  if (id === 'pods') {
```

Comment:
This is a good question. I had the console.debug because users could quickly get a print of all the tables processed by the example plugin for the view they're at.
I think it's useful to map IDs -> tables-in-views, but let's do that in another PR later if that's the best approach.
I think anyone developing a plugin using this processor will just check which names do those tables have dynamically anyway.